ksql-engine/src/test/java/io/confluent/ksql/function/udf/TestUdf.java

```diff
@@ -38,4 +40,11 @@ public String doStuffLongLongString(final long arg1, final long arg2, final Stri
   public String doStuffLongVarargs(final long... longs) {
     return "doStuffLongVarargs";
   }
+
+  @Udf(description = "returns the value of 'A'")
+  public String doStuffStruct(
+      @UdfParameter(schema = "STRUCT<A VARCHAR>") final Struct struct
```

Comment:
I will add a test. This is functionality that can always be added later, but can never be removed - and since this requires extra logic comparing `Schema#fields()` (the default `.equals()` is order sensitive) I'd rather leave it that way.

Off the top of my head, I know that AVRO does care about field ordering so it might be better to delay this and give it some more thought before implementing it 🤔

Reply:
Added it in `UdfLoaderTest`